### PR TITLE
ONEMPERS-306: HdcpProfile: returning local string c_str() corrected

### PR DESCRIPTION
**Pull request by michalkucharczyk**

### CODE REVIEW

**Comment by anand-ky** [comment]
@apatel859 - Can you pls check where this getHdcpReasonStr() is being used. Don't see it within RDKServices.

File: HdcpProfile/HdcpProfile.cpp [hunk 1]
```diff
@@ -44,7 +44,7 @@ namespace WPEFramework
     namespace Plugin
     {
         SERVICE_REGISTRATION(HdcpProfile, 1, 0);
-        static const char* getHdcpReasonStr (int eHDCPEnabledStatus);
+        static std::string getHdcpReasonStr (int eHDCPEnabledStatus);
```
Comment (apatel859) [inline HdcpProfile/HdcpProfile.cpp:47]: not used so remove it everwhere

**Comment by anand-ky** [comment]
@michalkucharczyk - good catch on returning local string. This API is not used. Can you help remove this as part of this PR.

**Review by anand-ky — APPROVED** [review]

**Comment by pradeeptakdas** [comment]
Internal ticket for tracking https://ccp.sys.comcast.net/browse/RDKCOM-2859

**Pull request merged**